pretty_midi/pretty_midi.py

```diff
@@ -1623,3 +1892,36 @@ def semitones_to_pitch_bend(semitones, semitone_range=2.):
             MIDI pitch bend amount, in [-8192, 8191]
     '''
     return int(8192*(semitones/semitone_range))
+
+
+def midi_key_to_key_name(key_signature_event):
```

Comment:
So, an assumption of `pretty_midi` is that someone using it will never have to use any functionality from the `midi` module.  The conversion functions I made are mostly general-purpose convenience.  Since this is really only used within `_load_metadata`, can we put this in there instead?